request.go

```diff
@@ -244,6 +244,19 @@ func unmarshalNode(data *Node, model reflect.Value, included *map[string]*Node)
 				continue
 			}
 
+			if v.Kind() == reflect.String {
+				stringValue := v.Interface().(string)
+
+				// The field may or may not be a pointer to a string; handle both cases
+				if fieldValue.Kind() == reflect.Ptr {
+					fieldValue.Set(reflect.ValueOf(&stringValue))
```

Comment:
Ah, perhaps I misunderstood you. In the case of a `*string`, zero value is going to be `nil`. However, as the code is now, it should not make any difference. The default value on the struct will be `nil`, and the `Set` call here will just be setting it to `nil` once again.